test/tests.js

```diff
@@ -0,0 +1,92 @@
+const db = require('../db')
+const expect = require('chai').expect
+const fs = require('fs')
+const expect = require('chai').expect
+
+before(function() {
+    fs.mkdirSync("./collections/soldierCollection1", { recursive: true })
+    fs.mkdir("./collections/soldierCollection2", { recursive: true })
+    fs.writeFile("./collections/soldierCollection1/1.json", JSON.stringify(soldierObj))
+})
+
+describe('MingoDB', function() {
```

Comment:
inside the dsecribe of mingo, have one describe for insert and one for find. other than that you're done